### ISSUE TITLE
The link supplied in a post's front matter isn't used

### ISSUE THREAD

**Issue by TJScalzo** [tholman/inspiring-online#185]
I just noticed that the link that you specify in the front matter of a post isn't used in any way. One would expect that the image would lead to the site the post is about but instead, the link assigned to it is a link to the post. In fact, clicking either the title or the image when on a post's page will cause the page to simply reload. I feel like this should be changed in some way. Let me know what you think.

**Comment by JohJakob** [comment]
@TJScalzo Yeah, we should change it. Maybe, the title could point to the post and the image to the website. But how should we communicate this to the reader. It's a bit confusing.

**Comment by TJScalzo** [comment]
@JohJakob Well a random idea I just had: Hovering over the image makes text appear that says "Go to the site" or something like that. That could possibly work.

**Comment by tholman** [comment]
Thats weird. Originally, the main page would redirect to the post, but then
on the post, the image and main text would link out to the site... must
have broke somewhere along the line.

On Sat, Apr 30, 2016 at 12:24 PM, Tim Scalzo <notifications@github.com>
wrote:

> @JohJakob <https://github.com/JohJakob> Well a random idea I just had:
> Hovering over the image makes text appear that says "Go to the site" or
> something like that. That could possibly work.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/tholman/inspiring-online/issues/185#issuecomment-215976774>
>



-- 
Tim Holman
tholman.com

**Comment by JohJakob** [comment]
@TJScalzo @tholman I'll look into it. I also like @TJScalzo's idea with the overlay.

**Comment by Haroenv** [comment]
The overlay would be a great case for 

```css
-webkit-backdrop-filter: blur(5px);
```

as a progressive enhancement

**Comment by TJScalzo** [comment]
I looked back at the history of that document and I believe it's actually _my_ fault that this problem happened. [When I turned the if statements into elsifs](https://github.com/tholman/inspiring-online/commit/0681a9d07b0a55d1aa10fce0ec6fd911de7c3caf#diff-663f387b6a1a407ab38de055a12bc7c8R26) I believe I copied [what I made for the home page](https://github.com/tholman/inspiring-online/commit/0681a9d07b0a55d1aa10fce0ec6fd911de7c3caf#diff-eacf331f0ffc35d4b482f1d15a887d3bR19) and  forgot to change `post.url` to `page.link` like the original was. I apologize for any inconveniences I might have caused, @tholman.
